src/modules/Common/containers/Container.module.css

```diff
@@ -92,10 +92,10 @@
 
   &.container__isGray {
     background-color: var(--colorBlack200);
-
+    /* 
     & hr {
       border-color: var(--colorWhite);
-    }
+    } */
```

Comment:
Please erase code if it is not used instead of commenting it out, otherwise we don't know what is useful and what is not 🙂